etc/onespan-sign-node.api.md

```diff
@@ -0,0 +1,521 @@
+## API Report File for "onespan-sign-node"
```

Comment:
This API report file is the basis for the markdown files documentation (to be added in separate PRs to keep this PR small)